extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1173,13 +1174,18 @@ export class AmpA4A extends AMP.BaseElement {
     const iframe = createElementWithAttributes(
         /** @type {!Document} */ (this.element.ownerDocument),
         'iframe', Object.assign(mergedAttributes, SHARED_IFRAME_PROPERTIES));
+    // TODO(keithwrightbos): noContentCallback?
+    this.xOriginIframeHandler_ = new AMP.AmpAdXOriginIframeHandler(this);
+    // Iframe is appended to element as part of xorigin frame handler init.
+    // Executive onCreativeRender after init to ensure it can get reference
+    // to frame but prior to load to allow for earlier access.
```

Comment:
I'm not sure that this really is "prior to load".  `xOriginIframeHandler_.init` will create a promise that attaches the iframe.  That promise begins execution...  "Soon".  (Whatever that means in ES6.)  I think the semantics are that it starts execution immediately and runs until the next thread yield.  As soon as the iframe is attached, it will start resource fetches.  It seems to me that it's conceivable that it will complete load before `onCreativeRender` gets to run.  (E.g., in the cache hit case.)  I'm not sure you can say anything about the sequencing of load vs `onCreativeRender` in this flow.

That said, I don't see that it *hurts* anything to do it this way, and you can conceivably be doing a network fetch while the `onCreativeRender` is firing.  So I guess my only concern is how you describe it in the docs, here and at `onCreativeRender`.